### DISCUSSION TITLE
Add a new repo to cake-contrib for Cake.SFTP

### DISCUSSION THREAD

**Discussion by JeroenVandezande** [cake-build/cake#4198] [Extension Q&A] [answered]
Hi,

I already have one plug-in in the cake-contrib org, I am working on a new plug-in atm, the repo exists: 
https://github.com/JeroenVandezande/Cake.SFTP

I am planning to finish and test this plug-in in the next days, what is the procedure for it to be moved to cake-contrib?

Best Regards,

Jeroen

**Comment by augustoproiete** [comment, marked as answer]
Hi @JeroenVandezande, here are the instructions for the transfer:

For GitHub, this is a multi-step operation due to how GitHub manages permissions for organizations.

1. Verify that I'm a member of the core Cake maintainer team on the official website: https://cakebuild.net/docs/team/
2. Transfer the repo to my GitHub user `augustoproiete` ([instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository))
3. I transfer it to the [Cake Contrib](https://github.com/cake-contrib) organization
4. ~You accept the invite to join the organization (you should have received an email)~ You're already in the Cake Contrib organization (through your [Cake.SharePoint](https://github.com/cake-contrib/Cake.SharePoint) addin)
5. I add you as admin in your repo, within the org

For NuGet, open the "Manage" package UI for the Cake.SFTP package on [nuget.org](http://nuget.org/) and add the user `cake-contrib` as owner

**Reply by JeroenVandezande** [reply]
I have transfered the repo to you and I also added cake-contrib to the Nuget Package

**Reply by augustoproiete** [reply]
Thanks @JeroenVandezande . Unfortunately I didn't accept the repo transfer fast enough, and Github expires the link after a few hours. Could you please send the transfer again?

<img width="601" alt="image" src="https://github.com/cake-build/cake/assets/177608/b6946c58-288a-4a57-8150-1c5a00880ed1">

**Reply by JeroenVandezande** [reply]
I started the transfer again

**Reply by augustoproiete** [reply]
@JeroenVandezande Thanks. You should be all set now: https://github.com/cake-contrib/Cake.SFTP
